### DISCUSSION TITLE
[Bug] 保留多个标签页时，标签页显示范围侵入窗口控制按钮位置

### DISCUSSION THREAD

**Discussion by sharkucn** [ywmoyue/biliuwp-lite#889] [bug]
### Bug 描述

如图
![屏幕截图 2024-10-26 195105](https://github.com/user-attachments/assets/5a8b14b8-5aad-45ba-a103-a2e837e8a7df)

此时标签栏向左滚动功能键被遮挡，无法点击。其余功能键由于窗口控制按钮未遮盖整个标签页栏所以可以点击。
![屏幕截图 2024-10-26 200705](https://github.com/user-attachments/assets/c361605c-55e9-436b-af6b-3ec5b855f6f9)

### 操作步骤

```plain text
1. 打开应用
2. 随意打开多个标签页
3. 可见问题发生
```

### 期望结果

限制标签页显示范围在窗口控制按键左侧

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 11 Dev (Build 27729.1000) (Canary Channel)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 备注

应用版本 4.7.6.0

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
这个我看看有没有办法解决吧，确实挺影响使用的

**Reply by ProJend** [reply]
按钮改到左侧，右侧加点边距
